.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,19 @@
+name: Pull request
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize]
+
+jobs:
+  tests:
+    name: Test
+    uses: swiftlang/github-workflows/.github/workflows/swift_package_test.yml@main
+    with:
+      linux_exclude_swift_versions: '[{"swift_version": "5.9"}, {"swift_version": "5.10"}]'
+      linux_pre_build_command: apt-get update && apt-get install -y libsqlite3-dev libncurses5-dev
```

Comment:
Can this be a reference to an external script file or does it have to be inline? Because you'll need something a bit longer to handle both Debian and RedHat lineages:

```sh
command -v apt >/dev/null 2>&1 && apt update && apt install -y libsqlite3-dev || (command -v yum >/dev/null 2>&1 && yum update -y && yum install -y sqlite-devel)
```